### PR TITLE
Set up new relic

### LINKED ISSUE #7: Add New Relic to Monitor Performance
Size: 1

[New Relic](http://newrelic.com/)

Acceptance Criteria
- Application is hooked up to New Relic for monitoring
- Instructions are documented in the README
- A stand up presentation of how to use New Relic should be given

-----------

The project will be expected to comfortably handle several hundreds of requests per minute. We will use New Relic to judge the following:

4: Average response time below 100ms with 600+ RPM
3: Average response time below 200ms with 400+ RPM
2: Average resposne time below 400ms with 300+ RPM
1: Average response time above 400ms, or unable to handle 300 RPM

### PR DESCRIPTION
**Pull request by toriejw**
New Relic has been tested locally, and needs to be tested in production (needs to be merged first).

fixes #7

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,6 @@
 !/log/.keep
 /tmp
 .byebug_history
+
+# Ignore application configuration
```
Comment (rrgayhart) [inline .gitignore]: Nice!

**Comment by rrgayhart** [comment]
Nice! My only question is, once you're off the project how should we access New Relic?

We should probably have a google drive file or something like that to store login information for things like New Relic - also, we should probably make sure it's set up on a New Relic account specifically for the project and not a person one, if it isn't already!

**Comment by rrgayhart** [comment]
Going to set up a basecamp in #21 !

**Comment by toriejw** [comment]
Awesome to have basecamp! Thanks :) 

I'm 99% sure it's not a person account (I tried it out a bit with Pat), but I am the "owner" which I can change to be anyone else (i.e. you).

**Comment by patwey** [comment]
I, Pat, can confirm said trial.

**Comment by rrgayhart** [comment]
LGTM! :ship: :shipit: :+1: :yellow_heart:

**Comment by worace** [comment]
:metal:

**Pull request merged**